src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -95,6 +94,11 @@ trait Backend {
    */
   def stdoutStderr(descriptor: WorkflowDescriptor, callName: String, index: ExecutionIndex): CallLogs
 
+  /**
+   * Provides a function that given a WdlFile, returns its hash.
```

Comment:
Actually I take that back a bit - if the comment sticks I do like returns over provides. THe other part of my confusion was that it was abstract in a trait, so i read 'provides' as in the trait provided this abstract function